docs/content/docs/configuration.md

```diff
@@ -51,6 +54,118 @@ status:
 default_decision: /http/example/authz/allow
 ```
 
+## Environment Variable Substitution
+> Only supported with the OPA runtime (`opa run`).
+
+Environment variables referenced with the `${...}` notation within the configuration
+will be replaced with the value of the environment variable.
+
+Example using `BASE_URL` and `BEARER_TOKEN` environment variables:
+```yaml
+services:
+  - name: acmecorp
+    url: ${BASE_URL}
+    credentials:
+      bearer:
+        token: "${BEARER_TOKEN}"
+
+discovery:
+  name: /example/discovery
+  prefix: configuration
+```
+The environment variables `BASE_URL` and `BEARER_TOKEN` will be substituted in when the config
+file is loaded by the OPA runtime.
+
+> If the variable is undefined then an empty string (`""`) is substituted. It will __not__
+raise an error.*
```

Comment:
Nit: trailing `*`?

Reply:
Remnant of swapping from `*text*` to `> text` 😄